.github/workflows/deploy-alpha.yml

```diff
@@ -28,9 +28,10 @@ jobs:
       - name: Version And Publish
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-          NEXT_PUBLIC_SEGMENT_KEY: JfTfmHaATChc4xueS7RcCBsixI71dJIJ
+          NEXT_PUBLIC_SM_UI_SEGMENT_KEY: Ng5oKJHCGpSWplZ9ymB7Pu7rm0sTDeiG
```

Comment:
are you sure the keys are not inverted ?
I'm stating this because we have much more events on SM-UI so I would think you created a new source for the init only.

Reply:
As we sent server side events on the first source we cannot send now client event through it. So we are force to do the opposite :)